### PR TITLE
Single-GPU CuPy Benchmarks

### PR DESCRIPTION
**Pull request by pentschev**
cc @mrocklin

### CODE REVIEW

**Comment by mrocklin** [comment]
cc @datametrician

File: _posts/2019-06-25-single-gpu-cupy-benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+* CUDA Toolkit 9.2
+
+
+General Performance
+-------------------
+
+I have generated a greph comprising various operations. Most of them perform
+well on a GPU using CuPy out of the box. See the graph below:
+
+<img src="/images/single-gpu-cupy-benchmarks.png">
```
Comment (mrocklin) [inline _posts/2019-06-25-single-gpu-cupy-benchmarks.md]: If you wanted to embed the Vegalite JSON output itself you might want to look at this section here: https://github.com/dask/dask-blog/pull/32/files#diff-eb9896a7fb2ebd01e20ad59828c95e83R45

Comment (mrocklin) [reply _posts/2019-06-25-single-gpu-cupy-benchmarks.md]: Also, this image here isn't added.  I'd be happy to copy-paste my version over though if that works for you (it plays nicely with the formatting of our layout).

Comment (pentschev) [reply _posts/2019-06-25-single-gpu-cupy-benchmarks.md]: Thanks for the link, I've updated it now.

**Comment by pentschev** [comment]
Graph preview

**Comment by pentschev** [comment]
Here's the (updated) graph preview that isn't rendered properly by GH's rich diff:
Rendered Graph

**Comment by pentschev** [comment]
Thanks @mrocklin for the "Increasing Performance" section, it has been merged here.

**Comment by mrocklin** [comment]
OK, I'm going to mark this as a draft and merge/publish.

**Pull request merged**